inst/include/distributions/distributions.hpp

```diff
@@ -15,6 +15,8 @@
 #define FIMS_DISTRIBUTIONS_HPP
 
 #include "functors/density_components_base.hpp"
-#include "functors/tmb_distributions.hpp"
+#include "functors/lognormal_lpdf.hpp"
```

Comment:
Not sure where, but somewhere it would be nice to define LPDF and LPMF for developers reading the code. This has no impact on users.

Reply:
And also these are not technically lpdf or lpmf since they're negatives. I don't care but something to think about. Many of the downstream uses of the likelihood components will be positive, such as cross validation, and so perhaps defining them as positive and then negating at the end is worth considering.

Reply:
I think this is a good point @Cole-Monnahan-NOAA, I will make a note to refactor this in the next sprint